tss-esapi/src/attributes/nv_index.rs

```diff
@@ -92,7 +92,7 @@ impl NvIndexAttributes {
         // TPMA_NV_AUTHWRITE, or TPMA_NV_POLICYWRITE shall be SET."
         if !(self.pp_write() | self.owner_write() | self.auth_write() | self.policy_write()) {
             error!(
-                "Non of the attributes PPWRITE, OWERWRITE, AUTHWRITE, POLICYWRITE have been set"
+                "Non of the attributes PPWRITE, OVERWRITE, AUTHWRITE, POLICYWRITE have been set"
             );
```

Comment:
I do not think this is correct. I think it should be OWNERWRITE

Reply:
:facepalm: yep, that's right. Adjustin'...